public/javascripts/data/paragraph.js

```diff
@@ -2,19 +2,26 @@ const Database = require('better-sqlite3');
 
 
 // Function writes paragraph to the database
-exports.insertParagraph = (paragraph) => {
+exports.insertParagraph = (paragraph, res, req) => {
```

Comment:
Remove `req` and `res` as arguments - pass only the objects you need. Don't let your network layers (routing/controllers) bleed into database layers